packages/kbn-dev-cli-runner/src/help.test.ts

```diff
@@ -173,10 +169,6 @@ describe.skip('getHelpForAllCommands()', () => {
 
         Global options:
           --global-flag      some flag that applies to all commands
-          --verbose, -v      Log verbosely
-          --debug            Log debug messages (less than verbose)
-          --quiet            Only log errors
-          --silent           Don't log anything
```

Comment:
Dropping these doesn't "feel" right.  
I'll timebox an investigation into this soon.